src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,56 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_SIZE = 10;
+    private static final int DEFAULT_ARRAY_INDEX = -1;
+    private int count = 0;
+    private K[] keyArray = (K[]) new Object[ARRAY_SIZE];
+    private V[] storageArray = (V[]) new Object[ARRAY_SIZE];
```

Comment:
It would be best if you initialize these fields in the constructor
```suggestion
    private K[] keyArray;
    private V[] storageArray;
```